lte/gateway/c/core/oai/tasks/nas5g/src/SmfMessage.cpp

```diff
@@ -84,16 +88,15 @@ int SmfMsg::SmfMsgDecodeMsg(SmfMsg* msg, uint8_t* buffer, uint32_t len) {
   int decode_result = 0;
   int header_result = 0;
 
-  MLOG(MDEBUG) << "SmfMsgDecodeMsg:" << std::endl;
+  OAILOG_DEBUG(LOG_NAS5G, "Decoding SMF message");
```

Comment:
Convert the Message type to string so that in field level the logs can be avoided.